client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,107 @@
+/** @format */
+
+import React from 'react'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import IconButton from '@material-ui/core/IconButton'
+import Typography from '@material-ui/core/Typography'
+import InputBase from '@material-ui/core/InputBase'
+import { fade, makeStyles } from '@material-ui/core/styles'
+import SearchIcon from '@material-ui/icons/Search'
+import Logo from './asset/logo.png'
```

Comment:
could you please move the logo image out to the static folder, we'll be saving any static files images in that folder